### DISCUSSION TITLE
Is HyperNetX still actively being supported?

### DISCUSSION THREAD

**Discussion by cgyurgyik** [pnnl/HyperNetX#163] [Q&A]
Hi! I am looking to visualize some hypergraph work via a lightweight Python library such as your own. I noticed a version update last week, but the first Colab I tried did not work:

```py
ModuleNotFoundError: No module named 'hypernetx'
```

...and there are some stale documentation links. Is there still active support for this library?

**Comment by brendapraggastis** [comment]
Hi Chris,

Not sure if you got a reply from us, but yes HNX is alive and well. I will check into Colab issue, but you should be able to pip install HNX. If you are still having trouble please let us know.

Brenda

From: Chris Gyurgyik ***@***.***>
Date: Thursday, October 3, 2024 at 11:03 AM
To: pnnl/HyperNetX ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [pnnl/HyperNetX] Is HyperNetX still actively being supported? (Discussion #163)
Check twice before you click! This email originated from outside PNNL.


Hi! I am looking to visualize some hypergraph work via a lightweight Python library such as your own. I noticed a version update last week, but the first Colab I tried did not work:

ModuleNotFoundError: No module named 'hypernetx'

...and there are some stale documentation links. Is there still active support for this library?
